Doc/library/importlib.metadata.rst

```diff
@@ -127,7 +128,7 @@ a collection of all ``EntryPoint`` objects with ``names`` and ``groups``
 attributes for convenience::
 
     >>> sorted(eps.groups)  # doctest: +SKIP
-    ['console_scripts', 'distutils.commands', 'distutils.setup_keywords', 'egg_info.writers', 'setuptools.installation']
+    {'array_api', 'console_scripts', 'gui_scripts', 'pytest11', 'sphinx.html_themes'}
```

Comment:
Previously, this example used the actual entry point groups that would be presented by setuptools and no other project. With this change, it now presents entry point groups for some of setuptools (or maybe another build tool?), pytest, sphinx, and whatever solicits `array_api`). I feel like the previous example showing what would happen in a clean virtualenv (with setuptools installed) was cleaner. I'd be okay with providing other examples, but if we're going to tweak the list here, we should improve the clarity and not bring in arbitrary examples that happen to exist in one developer's environment.

Reply:
This was just a selection from e.g. NumPy, PyTest, Sphinx, etc to demonstrate potential entry points, but I take your point.